plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveMetadata.java

```diff
@@ -1327,8 +1327,8 @@ public HiveOutputTableHandle beginCreateTable(ConnectorSession session, Connecto
         }
```

Comment:
Why 1 is so special? What prevents us to support `skip_header_line_count=2`?

Reply:
It is special because it is mentioned here https://github.com/trinodb/trino/issues/6132 ;)

Jokes apart, my understanding(please keep in mind I'm new to this community so I may be totally wrong) is that this parameter is troublesome. It makes trino skip some rows on read when it may not be expected to. When it is set to one it can be compensated with header line so no actual data rows are skipped on read. When it is set to 2 and higher values I don't know what would be the expected behaviour.

Reply:
Technically for 2 we could also pad the file with something. But it is hard to come up with a reasonable padding. For `=1` we pad with header with column names. For `>1` IDK. Also I kinda believe that in the real world values >1 do not exist :)